### PR TITLE
Fix accommodation supplement rate & rebate when lodging

### PR DESCRIPTION
**Pull request by bonjourmauko**
* Tax and benefit system evolution.
* Impacted periods: from 26/11/2018.
* Impacted areas: `accommodation_supplement`
* Details:
  - Fix accommodation supplement rate & rebate when the accommodation type is lodging
  - Normally, a %62 rate is to be applied in this particular case, but %162 was being applied

- - - -

These changes:

- Fix or improve an already existing calculation.

### CODE REVIEW

File: openfisca_aotearoa/variables/acts/social_security/accommodation_supplement/rate.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    documentation = """TODO"""
+    entity = entities.Person
+    value_type = float
+    default_value = 0
+    definition_period = periods.DateUnit.WEEK
+
+    def formula_2018_11_26(people, period, _params):
+        # TODO: move to parameter
+        accommodation_type = people("accommodation_type", period)
+        return (
```
Comment (verbman) [inline openfisca_aotearoa/variables/acts/social_security/accommodation_supplement/rate.py]: The following two lines appear to ignore that this rate also applies to boarders.
Social Security Act 2018, s 65 subs 2 definition of “accommodation costs” (c). "who is a boarder or lodger in any premises".
There is an AccommodationType.board enum value.

File: openfisca_aotearoa/variables/acts/social_security/accommodation_supplement/rate.py [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+from openfisca_core import periods, variables
+
+from openfisca_aotearoa import entities
+from openfisca_aotearoa.variables.demographics.housing import AccommodationType
+
+
+class accommodation_supplement__rate(variables.Variable):
+    label = "TODO"
+    reference = "TODO"
```
Comment (verbman) [inline openfisca_aotearoa/variables/acts/social_security/accommodation_supplement/rate.py]: The reference is: 
https://www.legislation.govt.nz/act/public/2018/0032/latest/whole.html#LMS28903

File: openfisca_aotearoa/tests/social_security/accommodation_supplement/2018/exclusion.yaml [hunk 3]
```diff
@@ -1,8 +1,9 @@
...
-    65 Accommodation Supplement -
-    66 Social housing exclusion
-    (a) Rent
+    Subpart 10 Accommodation Supplement -
+    66  Social housing exclusion -
+    (1) Social housing exclusion -
+        (a) Rent
   period: 2018-11-26
   input:
     accommodation_type:
```
Comment (verbman) [inline openfisca_aotearoa/tests/social_security/accommodation_supplement/2018/exclusion.yaml:9]: What is the purpose of `under_the_sun`? It's not a valid AccommodationType value, I note further in this file is `under_the_moon`. I'm not entirely sure why OpenFisca accepts this.

File: openfisca_aotearoa/tests/social_security/accommodation_supplement/2018/exclusion.yaml [hunk 4]
```diff
@@ -87,27 +92,31 @@
...
-    65 Accommodation Supplement -
-    66 Social housing exclusion
-    (c) Residential care
+    Subpart 10 Accommodation Supplement -
+    67  Other funding exclusion -
+        (c) Residential care
   period: 2018-11-26
   input:
     accommodation_type:
       2022-W01: [residential_care, under_the_moon]
```
Comment (verbman) [inline openfisca_aotearoa/tests/social_security/accommodation_supplement/2018/exclusion.yaml:111]: What is the purpose of `under_the_moon`? It's not a valid AccommodationType value, note the other comment regarding `under_the_sun`. I'm not entirely sure why OpenFisca accepts this.

Comment (bonjourmauko) [reply openfisca_aotearoa/tests/social_security/accommodation_supplement/2018/exclusion.yaml:111]: Hey! Yes, these are arbitrary values.

**Review by verbman — APPROVED** [review]

**Pull request merged**